shardingsphere-infra/shardingsphere-infra-common/src/main/java/org/apache/shardingsphere/infra/datasource/pool/metadata/impl/HikariDataSourcePoolMetaData.java

```diff
@@ -71,6 +71,8 @@ private void buildPropertySynonyms() {
     }
     
     private void buildTransientFieldNames() {
+        transientFieldNames.add("username");
+        transientFieldNames.add("password");
```

Comment:
The attributes of `username` and `password` should not add here.
They are not transient fields, they just belong to connection properties.